src/Models/CreatePaymentRequest/PaymentCheckout.cs

```diff
@@ -5,24 +5,27 @@ namespace Dynamicweb.Ecommerce.CheckoutHandlers.DibsEasyCheckout.Models.CreatePa
 [DataContract]
 internal sealed class PaymentCheckout
 {
-    [DataMember(Name = "url")]
+    [DataMember(Name = "url", EmitDefaultValue = false)]
     public string Url { get; set; }
 
-    [DataMember(Name = "returnUrl")]
+    [DataMember(Name = "returnUrl", EmitDefaultValue = false)]
     public string ReturnUrl { get; set; }
 
     [DataMember(Name = "termsUrl")]
```

Comment:
Should this not also have EmitDefaultValue?

Reply:
Terms url is required one
https://developer.nexigroup.com/nexi-checkout/en-EU/api/payment-v1/#v1-payments-post-body-checkout-termsurl